aimmdb/utils.py

```diff
@@ -37,18 +38,20 @@ def read_group(g, jsoncompat=False):
     return out
 
 
-_ELEMENT_DATA = None
-
-
+@lru_cache(maxsize=2)
```

Comment:
why `maxsize=2`? Since the function has no arguments I think it can just be 1.

Reply:
I figured just to be sure.

No you're actually right though lol